src/components/elements/background-image.tsx

```diff
@@ -44,7 +44,7 @@ export const Background = ({
 
     return (
         <StyledBackground Tag="div" style={style} $dark={dark} {...bg_image} {...props}>
-            {children}
+            {children ? children : <></>}
```

Comment:
Can we write this like this? 
`{children && children}`

Reply:
I've tried this and I got this error:
![ error](https://user-images.githubusercontent.com/93753441/182143239-75607b8e-1dfa-4ea5-91ae-797e27c04fdd.png)

Reply:
alrighty